pkg/storaged/stratis-panel.jsx

```diff
@@ -131,18 +137,32 @@ export function create_stratis_pool(client) {
                                                  if (phrase === "")
                                                      return _("Passphrase cannot be empty");
                                              },
-                                             visible: vals => vals.encrypt.on
+                                             visible: vals => vals.encrypt_pass.on
                                          }),
                                PassInput("passphrase2", _("Confirm"),
                                          {
                                              validate: function (phrase2, vals) {
                                                  if (phrase2 != vals.passphrase)
                                                      return _("Passphrases do not match");
                                              },
-                                             visible: vals => vals.encrypt.on
+                                             visible: vals => vals.encrypt_pass.on
                                          })
                            ]
                        }),
+            CheckBoxes("encrypt_tang", "",
+                       {
+                           visible: () => client.features.stratis_crypto_binding,
+                           fields: [
+                               { tag: "on", title: _("Encrypt data with a keyserver") }
```

Comment:
Should this perhaps spell out "Tang" in the name explicitly? So far I know "keyservers" from PGP/GPG, and it's a very generic term.